src/components/form-elements/_text-input.scss

```diff
@@ -1,148 +1,137 @@
-$inputHeight: rhythm(3 / 2);
-$inputLargeHeight: rhythm(8 / 3);
-$inputSmallHeight: rhythm(5 / 4);
+$inputHeight: 40px;
+$inputLargeHeight: 56px;
 
-$inputBackground: $white;
+$inputBackground: $graySecondaryLightest;
 $inputTextColor: $black;
 $inputFontSize: 16px; // fix zooming to input on focus in iOS devices
-$inputLargeFontSize: fontSize(standout);
+$inputLargeFontSize: 18px;
 $inputPlaceholderTextColor: $graySecondary;
-$inputFocusValidBorderColor: $mintPrimaryDark;
-$inputFocusInvalidBorderColor: $peachPrimaryDark;
-$inputStandardBorderRadius: 9px;
-$inputPlaceholderFontSize: 16px;
-
-$inputSmallBorderRadius: 7px;
-$inputSmallPlaceholderFontSize: 12px;
-
-$inputLargeBorderRadius: 13px;
-
-$searchInputTransparentColor: rgba($white, 0.4);
-$searchInputTransparentColorAlt: rgba($white, 0.8);
-
-$searchInputBoxShadow: 0 1px 4px 0 rgba(0, 0, 0, 0.2);
+$inputFocusValidBorderColor: $mintPrimary;
+$inputFocusInvalidBorderColor: $peachPrimary;
+$inputStandardBorderRadius: 20px;
+$inputLargeBorderRadius: 28px;
 
 $includeHtml: false !default;
 
 @if ($includeHtml) {
-
   .sg-input {
     @include component();
-    background: $inputBackground;
-    border: 2px solid $graySecondaryLight;
+    background: $inputBackground; 
+    border: 2px solid $inputBackground;
     border-radius: $inputStandardBorderRadius;
     color: $inputTextColor;
     font-size: $inputFontSize;
-    padding: 0 gutter(1 / 2);
+    font-weight: normal;
+    padding: 0 spacing(s);
     height: $inputHeight;
     appearance: none;
-    transition: border 300ms;
+    transition: background 100ms, border 100ms;
 
     &:focus,
-    &:active {
+    &:active,
+    &:focus:hover,
+    &:active:hover {
       outline: none;
-      box-shadow: $elevatedComponentShadow;
-      border: 2px solid transparent;
       background-color: $white;
-
-      &::placeholder {
-        visibility: hidden;
-      }
+      border: 2px solid $graySecondaryLight;
     }
 
     &::placeholder {
       color: $inputPlaceholderTextColor;
-      font-size: $inputPlaceholderFontSize;
-      font-weight: $fontWeightBold;
+      font-size: $inputLargeFontSize;
+      font-weight: normal;
       position: relative;
       opacity: 1;
+      visibility: visible;
     }
 
-    &--light {
-      transition: background 0.3s ease-out, box-shadow 0.3s ease-out;
-      background: $searchInputTransparentColor;
-      box-shadow: $searchInputBoxShadow;
-      border-radius: $borderRadiusDefault;
-      border: none;
+    &:hover {
+      background-color: mix($inputBackground, $graySecondaryLight, 80%);
+      border: 2px solid mix($inputBackground, $graySecondaryLight, 80%);
+    }
+  
+    &--white {
+      background: $white;
+      border: 2px solid $white;
+      border-radius: $inputStandardBorderRadius;
 
-      &::placeholder {
-        color: $white;
-        font-weight: $fontWeightBold;
+      &:hover {
+        background-color: mix($white, $graySecondaryUltraLight, 12%);
```

Comment:
where does `12%` comes from? is the final color part of our palette?

Reply:
it's not, this is the same pattern as we use for `Button`. For filled element, we use 80% mix and for light version 12%. I will try to describe it in some part of the README.